packages/forms-web-app/src/dynamic-forms/has-questionnaire/journey.js

```diff
@@ -2,9 +2,11 @@ const { questions } = require('./questions');
 const { Journey } = require('../journey');
 const { Section } = require('../section');
 
+const baseHASUrl = '/manage-appeals/questionnaire';
+
 class HasJourney extends Journey {
-	constructor(response) {
-		super(response);
+	constructor(journeyId, response) {
```

Comment:
The reference ID should already be included in the `response` object so shouldn't need to be included separately. The response object should look like the following:
```
{
    referenceId: "123456",
    formId: "has-questionnaire",
    startDate: "2023-08-01T14:32:19Z",
    updateDate: "2023-08-02T16:09:47Z",
    answers: {
        "green-belt": "no",
        "impacts-listed-building": "yes",
        "listed-buildings": [
            {
                name: "Stone bridge to the rear of Greendale Farm",
                grade: "II",
                listingId: "1234567"
            }
        ]
    }
}
```

Reply:
I moved it into the constructor instead of each function as we will always know it upfront
Could move to response but I guess that means we will always need a blank response when we create a journey
I'll rename to referenceId to be consistent